packages/vaadin-crud/src/vaadin-dialog-layout.js

```diff
@@ -150,6 +150,9 @@ class DialogLayout extends ElementMixin(ThemableMixin(PolymerElement)) {
       this._ensureChildren();
     }
 
+    // TODO: A temporary hack as far as `vaadin-dialog` doesn't support the Polymer Template API anymore.
+    this.$.dialog.$.overlay.template = this.$.dialog.querySelector('template');
```

Comment:
@tomivirkki I wonder if it would work when there `@vaadin/vaadin-template-renderer` is imported.

I pass `template` to `overlay` but at that time the template probably has been already handled by the template renderer and passed to `overlay` as a renderer function.

Reply:
Maybe if it would work if the `<template>` element is moved from inside the `<vaadin-dialog>` next to it.

Reply:
Moved.